packages/node/src/edge-functions/edge-handler.ts

```diff
@@ -69,13 +69,28 @@ async function compileUserCode(
 
       // user code
       ${compiledFile.text};
+      const userEdgeHandler = module.exports.default;
```

Comment:
**question (non-blocking):** I follow that the ref to `module.exports.default` was moved from the template into here, but I don't get why it needed to be moved here. In theory, `registerFetchListener()` would be eval'd in the same context and thus resolve the same `module.exports.default`.

Reply:
The `module.exports.default` was looking for the user's code from `compiledFile`. I put it right after the `compiledFile` interpolation to make that relationship visual.

The `edgeHandlerTemplate` is not related to this `moduled.exports.default`. The template comes later.